Jenkinsfile

```diff
@@ -1,8 +1,4 @@
-//def buildConfiguration = buildPlugin.recommendedConfigurations()
-
-def buildConfiguration = [
-  [platform: 'windows', jdk: '11'],
-  [platform: 'linux',   jdk: '11'],
-]
-
-buildPlugin(configurations: buildConfiguration, timeout: 90)
+buildPlugin(useContainerAgent: true, configurations: [
+  [platform: 'linux', jdk: 17],
```

Comment:
```suggestion
  [platform: 'linux', jdk: 17],
  [platform: 'linux', jdk: 11],
```

Reply:
I'd preferred to not remove Linux Java 11 build and test

Reply:
Inconsistent with the archetype, but OK